### ISSUE TITLE
[MU4 Issue] Keyboard Shortcut Creation Not Accessible Using Screen Reader

### ISSUE THREAD

**Issue by NoahTCarver** [musescore/MuseScore#16157]
**Describe the bug**
It is impossible to save a keyboard shortcut created in MuseScore 4 using a screen reader.

**To Reproduce**
1. Launch screen reader of choice (confirmed on Windows and Linux)
2. Open 'Preferences'
3. Select 'Shortcuts'
4. Find a shortcut to edit and press 'space'
5. Type shortcut
6. Reach dead end; not possible to navigate to save button. Only possible recourse for user is to press escape, exiting the shortcut window and discarding the input.

**Expected behavior**
The user should be able to navigate to other buttons in the window, most important among them being the save button.

**Platform information**
- OS: Windows 11, Linux
- Screen readers: JAWS, NVDA, Orca

**Additional context**
This issue was also reported in the course of diagnosing another, unrelated set of issues involving screen reader accessibility:
Also, it should work now to assign a keyboard shortcut to RNA - except unfortunately the shortcut dialog is currently not accessible, it's impossible to save the shortcut and close the dialog using the keyboard alone.

_Originally posted by @MarcSabatella in https://github.com/musescore/MuseScore/issues/16020#issuecomment-1406745262_

**Comment by msviolafangirl** [comment]
Thank you Noah, I reported the same issue at #17923. It is a duplicate so I will close that issue.

**Comment by shubham-shinde-442** [comment]
Can I give this issue a try?

**Comment by DmitryArefiev** [comment]
Implemented in #22036
